runtime/mainnet/src/config/monetary.rs

```diff
@@ -0,0 +1,375 @@
+use pop_runtime_common::UNIT;
+
+use crate::{
+	parameter_types, AccountId, Balance, Balances, ConstU32, ConstU8, ConstantMultiplier,
+	ResolveTo, Runtime, RuntimeEvent, RuntimeFreezeReason, RuntimeHoldReason,
+	SlowAdjustingFeeUpdate, Ss58Codec, System, VariantCountOf, EXISTENTIAL_DEPOSIT,
+};
+
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	// src: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/constants/src/polkadot.rs#L70
+	(items as Balance * 20 * UNIT + (bytes as Balance) * 100 * fee::MILLI_CENTS) / 100
+}
```

Comment:
Mainnet is using its own deposit function, different from devnet and testnet.

Reply:
(Reason given): Basically we were not depending on that crate yet. Figured that having the fn locally would be more explicit and easier to identify